metadata_store_scripts/V2025.07.22.0__add_generate_masking_parameters_procedure.sql

```diff
@@ -0,0 +1,391 @@
+        CROSS APPLY
+            OPENJSON (r.assigned_algorithm)
+            WITH (
+                [key_column] NVARCHAR(255) '$.key_column',
+                [conditions] NVARCHAR(MAX) '$.conditions' AS JSON
+            ) AS x
+        CROSS APPLY
+            OPENJSON (x.[conditions])
+            WITH (
+                [condition_alias] NVARCHAR(255) '$.alias',
+                [algorithm] NVARCHAR(255) '$.algorithm'
+            ) AS c
+        WHERE
+            ISJSON(r.assigned_algorithm) = 1
+            AND JSON_VALUE(r.assigned_algorithm, '$.key_column') IS NOT NULL
+            AND c.[condition_alias] = @filter_key
+            AND @filter_key <> ''
+    ),
+
+    -- standard_algorithm_handling - Handle non-conditional algorithms
+    standard_algorithm_handling AS (
+        SELECT
+            r.*,
+            r.assigned_algorithm AS resolved_algorithm
+        FROM ruleset_computed AS r
+        WHERE
+            ISJSON(r.assigned_algorithm) = 0
+            OR @filter_key = ''
+    ),
+
+    -- algorithm_resolution - Combine conditional and standard algorithms
+    algorithm_resolution AS (
+        SELECT
+            dataset,
+            specified_database,
+            specified_schema,
+            identified_table,
+            identified_column,
+            identified_column_type,
+            identified_column_max_length,
+            row_count,
+            ordinal_position,
+            encoded_column_name,
+            resolved_algorithm AS assigned_algorithm
+        FROM conditional_algorithm_extraction
+
+        UNION ALL
+
+        SELECT
+            dataset,
+            specified_database,
+            specified_schema,
+            identified_table,
+            identified_column,
+            identified_column_type,
+            identified_column_max_length,
+            row_count,
+            ordinal_position,
+            encoded_column_name,
+            resolved_algorithm AS assigned_algorithm
+        FROM standard_algorithm_handling
+        WHERE
+            resolved_algorithm IS NOT NULL
+            AND resolved_algorithm <> ''
+            AND ISJSON(resolved_algorithm) = 0
+    ),
+
+    -- ruleset_with_types - Join with ADF types, calculate column width estimates
+    ruleset_with_types AS (
+        SELECT
+            ar.*,
+            f.adf_type,
+            f.treat_as_string,
+            CASE
+                WHEN ar.identified_column_max_length > 0 THEN ar.identified_column_max_length + 4
+                ELSE @column_width_estimate
+            END AS column_width_estimate
+        FROM algorithm_resolution AS ar
+        INNER JOIN ruleset_computed AS f
+            ON ar.identified_column = f.identified_column
+        WHERE (
+            f.[key_column_name] IS NULL
+            OR f.identified_column <> f.[key_column_name]
+        )
+    ),
+
+    -- generate_mask_parameters - Create core masking parameters with fallback defaults
+    generate_mask_parameters AS (
```

Comment:
issue: can you please fix the casing of these parameters so that they are consistent?